source/_patterns/00-atoms/inline/date/date.twig

```diff
@@ -1,6 +1,6 @@
-{%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['date-time']) }) -%}
+{%- set attributes = attributes|default({})|merge({datetime: date|date("Y-m-d")}) -%}
```

Comment:
This is what I meant in https://github.com/libero/pattern-library/pull/75#discussion_r272064073.